mod/01_filter/set/abstract/filter.rb

```diff
@@ -184,27 +221,85 @@ def multiselect_filter type_codename, label=nil
 
   view :name_formgroup do |args|
     name = args[:name] || "name"
-    text_filter name, title: "Name"
+    text_filter name, title: "Keyword"
   end
 
   view :project_formgroup do
-    select_filter :project
+    select_filter :project, "asc"
   end
 
   view :year_formgroup do
-    select_filter :year
+    select_filter :year, "desc"
   end
 
-  view :topic_formgroup do
-    select_filter :wikirate_topic
+  view :wikirate_topic_formgroup do
+    select_filter :wikirate_topic, "asc"
   end
 
   view :metric_formgroup do
-    select_filter :metric
+    select_filter :metric, "asc"
+  end
+
+  view :wikirate_company_formgroup do
+    select_filter :wikirate_company, "asc"
   end
 
-  view :company_formgroup do
-    select_filter :wikirate_company
+  view :metric_value_formgroup do
+    options = {
+      "Exists" => "exists",
+      "None" => "none",
+      "Edited in last hour" => "last_hour",
+      "Edited today" => "today",
+      "Edited this week" => "week",
+      "Edited this month" => "month"
+    }
+    simple_select_filter "value", options, (Env.params["value"] || "exists")
+  end
+
+  view :metric_type_formgroup do
+    type_card = Card["metric_type"]
+    options = Card.search type_id: type_card.id, return: :name, sort: "name"
+    checkbox_formgroup "Type", options
+  end
+
+  def checkbox_formgroup title, options, default=nil
+    key = title.to_name.key
+    param = Env.params[key] || default
+    checkboxes = options.map do |option|
+      checked = param.present? && param.include?(option.downcase)
+      %(<label>
+        #{check_box_tag("#{key}[]", option.downcase, checked) + option}
+      </label>)
+    end
+    formgroup title, checkboxes.join("")
+  end
+
+  # def check_box_params option, param
+  #   checked = param.present?
+  #   result =
+  #     if option.is_a? Array
+  #       [option[0], option[1]]
+  #     else
+  #       [option, option.downcase]
+  #     end
+  #   result.push(checked && param.include?(result[1].downcase))
+  # end
+
+  view :research_policy_formgroup do
+    options = type_options :research_policy
+    checkbox_formgroup "Research Policy", options
+  end
+
+  view :metric_value_formgroup do
+    options = {
+      "Exists" => "exists",
+      "None" => "none",
+      "Edited in last hour" => "last_hour",
+      "Edited today" => "today",
+      "Edited this week" => "week",
+      "Edited this month" => "month"
```

Comment:
do all these options actually work?

(even if so, we can probably drop "edited in last hour" for now)